File: SAS Token/sas_token.js
const { BlobServiceClient } = require('@azure/storage-blob');

const storageAccountUrl = 'https://yourstorageaccount.blob.core.windows.net/';
const sasToken = 'your_sas_token_here';

const blobServiceClient = new BlobServiceClient(`${storageAccountUrl}?${sasToken}`);

const containerName = 'your-container-name';
const blobName = 'your-blob-name';

async function downloadBlob() {
    const containerClient = blobServiceClient.getContainerClient(containerName);
    const blobClient = containerClient.getBlobClient(blobName);
    const downloadBlockBlobResponse = await blobClient.download(0);
    console.log(await downloadBlockBlobResponse.blobBody());
}

downloadBlob().catch(console.error);

File: SAS Token/sas_token.py
from azure.storage.blob import BlobServiceClient

storage_account_url = "https://yourstorageaccount.blob.core.windows.net/"
sas_token = "your_sas_token_here"

blob_service_client = BlobServiceClient(
    account_url=storage_account_url, credential=sas_token
)

container_name = "your-container-name"
blob_name = "your-blob-name"

blob_client = blob_service_client.get_blob_client(
    container=container_name, blob=blob_name
)

downloaded_blob = blob_client.download_blob().readall()
print(downloaded_blob)
